arbitrum-docs/node-running/how-tos/read-sequencer-feed.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "How to read the sequencer feed"
+todos:
+  - Follow convention and style guide
+  - Communicate "who this is for" and "under which scenarios this is useful".
+  - Set expectations - prior knowledge, what to expect
+  - Build supporting conceptual content
+  - Align on what we want to treat as proper nouns vs common nouns
+---
+
+Running an Arbitrum relay locally as a Feed Relay lets you subscribe to the Sequencer feed for real-time data as the Sequencer accepts and orders transactions off-chain. 
```

Comment:
This is a "copy & light edit" from the FAQ answer. This doesn't follow the conventions in our style & convention guide yet (that's beyond the intended scope of this PR)